migtests/tests/oracle-tests/ora-trunc-tests/ora-date-trunc-test/validate

```diff
@@ -17,9 +17,8 @@ def migration_completed_checks(tgt):
 		print(f"table_name: {table_name}, row_count: {got_row_count[table_name]}")
 		assert row_count == got_row_count[table_name]
 	
-	view_list = tgt.get_objects_of_type("VIEW", "test_mysql_views")
-	print("view_list:", view_list)
-	assert len(view_list) == 1
+	function_count = tgt.get_functions_count("public")
+	assert function_count == 2
```

Comment:
If object count is relatively small, it is better to make the check a bit stricter by checking even for the names. Consider something like:

```
function_names = tgt.list_function_names(schema="public")
assert set(function_names) == set(["func_name_1", "func_name_2"])
```

Reply:
Added the assertion now